go.mod

```diff
@@ -1,35 +1,83 @@
 module github.com/keikoproj/addon-manager
 
-go 1.16
+go 1.17
 
 require (
-	cloud.google.com/go v0.65.0 // indirect
 	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/argoproj/argo-workflows/v3 v3.2.6
 	github.com/go-logr/logr v0.4.0
-	github.com/go-logr/zapr v0.2.0 // indirect
-	github.com/google/go-cmp v0.5.2 // indirect
-	github.com/google/gofuzz v1.2.0 // indirect
-	github.com/google/uuid v1.1.2 // indirect
-	github.com/googleapis/gnostic v0.5.1 // indirect
-	github.com/imdario/mergo v0.3.11 // indirect
 	github.com/jinzhu/inflection v1.0.0
-	github.com/onsi/ginkgo v1.16.2
-	github.com/onsi/gomega v1.11.0
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/common v0.13.0 // indirect
-	github.com/spf13/cobra v1.0.0
-	go.uber.org/zap v1.15.0 // indirect
-	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a // indirect
-	golang.org/x/net v0.0.0-20201202161906-c7110b5ffcbb
-	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
-	gomodules.xyz/jsonpatch/v2 v2.1.0 // indirect
+	github.com/spf13/cobra v1.2.1
+	golang.org/x/net v0.0.0-20210614182718-04defd469f4e
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
-	k8s.io/api v0.19.11
-	k8s.io/apiextensions-apiserver v0.19.11
-	k8s.io/apimachinery v0.19.11
-	k8s.io/client-go v0.19.11
-	k8s.io/klog/v2 v2.3.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20200831175022-64514a1d5d59 // indirect
-	k8s.io/utils v0.0.0-20200821003339-5e75c0163111 // indirect
-	sigs.k8s.io/controller-runtime v0.6.5
+	k8s.io/api v0.21.5
+	k8s.io/apiextensions-apiserver v0.21.2
```

Comment:
Why is this one not `v0.21.5`?

Reply:
this is compitible with all the dependencies. 
there is no difference between 21.2 and 21.5